lib/srh/http/base_router.ex

```diff
@@ -85,6 +88,9 @@ defmodule Srh.Http.BaseRouter do
         {:not_authorized, message} ->
```

Comment:
Would it make sense to encode other errors the same way? (Maybe in a separate PR?)

Reply:
Haha, yes, I can do another PR for error overhaul and clean this all up